AdditionalCodingGuidelines.md

```diff
@@ -317,7 +322,7 @@ fun registerApplicationModules(modules: List<String>) =
 Since javadoc is used for generating documentation in the Frank!Doc, it's important to provide usable information in the class for Frank Developers.
 
 * Please provide documentation in the javadoc if there's been a (breaking) change and offer an example in the documentation. See [FixedResultPipe.java](https://github.com/frankframework/frankframework/blob/master/core/src/main/java/org/frankframework/pipes/FixedResultPipe.java) for instance.  
-* When using Code examples, please use `<pre>{@code ... }</pre>` blocks.
+* When using multiline Code examples, please use `<pre>{@code ... }</pre>` blocks.
 * When referring to classes, please use `{@link ClassName}`
-* When referring to a value, please use `{@value #VALUE}`
-* When referring to a literal, please use `{@literal null}`
+* When referring to a class value, please use `{@value #VALUE}`
+* When referring to a value (of a parameter, variable, etc), please use `{@literal null}`
```

Comment:
Perhaps you can add a clarification that the Frank!Doc treats this differently than regular JavaDoc so that the literal will stand out as a code literal?